docs/build/eps.md

```diff
@@ -175,15 +176,15 @@ These instructions are for the latest [JetPack SDK 5.1.2](https://developer.nvid
       export CUDACXX="/usr/local/cuda/bin/nvcc"
       ```
 
-3. Install the ONNX Runtime build dependencies on the Jetpack 5.1.2 host:
+3. Install the ONNX Runtime build dependencies on the Jetpack host:
 
    ```bash
    sudo apt install -y --no-install-recommends \
      build-essential software-properties-common libopenblas-dev \
      libpython3.8-dev python3-pip python3-dev python3-setuptools python3-wheel
    ```
 
-4. Cmake is needed to build ONNX Runtime. For ONNX Runtime 1.16, the minimum required CMake version is 3.26 (version 3.27.4 has been tested). This can be either installed by:
+4. Cmake is needed to build ONNX Runtime. For ONNX Runtime 1.18, the minimum required CMake version is 3.26 (version 3.27.4 has been tested). This can be either installed by:
```

Comment:
1.17?

Reply:
thanks. removed reference to specific ort version.